pos_geonames/static/src/js/pos_geonames_field.js

```diff
@@ -0,0 +1,35 @@
+odoo.define("pos_geonames_field.ClientDetailsEditInherit", function (require) {
+    "use strict";
+
+    const PosComponent = require("point_of_sale.PosComponent");
+    const { _t } = require("web.core");
+    const { getDataURLFromFile } = require('web.utils');
+    const Registries = require('point_of_sale.Registries');
+    var ClientDetailsEdit = require("point_of_sale.ClientDetailsEdit");
```

Comment:
```suggestion
    const ClientDetailsEdit = require("point_of_sale.ClientDetailsEdit");
```

`var` is not used anymore in modern js. stick to `const` (mostly) and `let` (when needed)

https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/
summary: `var` has some issues that hard to debug -- community stopped using it in favor of the other two